payments-ui-core/api/payments-ui-core.api

```diff
@@ -396,6 +396,30 @@ public final class com/stripe/android/ui/core/elements/EmptyFormSpec$Creator : a
 	public synthetic fun newArray (I)[Ljava/lang/Object;
 }
 
+public final class com/stripe/android/ui/core/elements/ExternalPaymentMethodSpec$$serializer : kotlinx/serialization/internal/GeneratedSerializer {
```

Comment:
I'm not sure how to get these to not show up in the public API. They are also present for SharedDataSpec, FormItemSpec, etc. Do you know how to hide these @jaynewstrom-stripe ?

Reply:
These are fine, these are all synthetic.